crashclustering_data/schema_version

```diff
@@ -0,0 +1 @@
+"1"
```

Comment:
Since this is generated automatically by download_data, can you add it to the .gitignore file?